src/coprocessor/dag/expr/mod.rs

```diff
@@ -219,6 +219,7 @@ impl Expression {
     }
 
     pub fn build(ctx: &mut EvalContext, mut expr: Expr) -> Result<Self> {
+        debug!("build expr:{:?}", expr);
```

Comment:
I prefer to give a more formal log, i.e. `Coprocessor DAG build expr: {:?}`. Anyway this is trivial. LGTM.